### PR TITLE
Feat : Home page + Theme page

### PR DESCRIPTION
**Pull request by Logan1x**

### CODE REVIEW

**Review by shrey27 — APPROVED** [review]

File: app.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+const modal = document.getElementById("modal");
+const modalBtn = document.querySelectorAll(".modalBtn");
+const closeBtn = document.getElementById("closeModal");
+
+modalBtnList = [...modalBtn];
+
```
Comment (thegeorgenikhil) [inline app.js]: You forgot to remove this variable

File: styles/themes.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+  color: var(--color-foreground);
+  border-radius: 0.3rem;
+  text-align: left;
+  width: 230px;
+  margin: 0.3rem;
+  height: auto;
+}
+.card:hover {
+  /* cursor: pointer; */
+}
```
Comment (thegeorgenikhil) [inline styles/themes.css]: commented code

**Review by thegeorgenikhil — APPROVED** [review]
Good to merge!

File: app.js (hunk 1 above)
Comment (Logan1x) [reply app.js]: I don't think I want to remove that, This is to open the model when any of the card quiz start button is clicked.

Comment (thegeorgenikhil) [reply app.js]: Didn't see that anywhere else in that js file. That's why👍🏻

Comment (Logan1x) [reply app.js]: Oh, thanks. I was trying to iterate through nodelist but wasn't able to so I found this on StackOverflow. Now when you pointed out, I saw that I haven't used it. Thanks again :)

**Comment by Logan1x** [comment]
Thanks @shrey27 and @thegeorgenikhil for review :)

**Pull request merged**